drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1193,6 +1193,9 @@ function DragAndDropBlock(runtime, element, configuration) {
             // Make item draggable using the mouse
             try {
                 $item.draggable({
+                    appendTo: $root.find('.drag-container'),
+                    helper: 'clone',
+                    scroll: false,
```

Comment:
You can remove this. I'll merge https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/133 as soon as the tests pass, then you should rebase on top of master to verify these changes still work (I already tested these changes together with https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/133, and it seems to work fine).